client/src/cht.rs

```diff
@@ -23,11 +23,10 @@
 //! root has. A correct proof implies that the claimed block is identical to the one
 //! we discarded.
 
-use hash_db;
 use codec::Encode;
 use sp_trie;
 
-use sp_core::{H256, convert_hash};
+use sp_core::{H256, convert_hash, self};
```

Comment:
```suggestion
use sp_core::{H256, convert_hash};
```

We use using Rust 2018!